### PR TITLE
Return all status in GUI

### PR DESCRIPTION
**Pull request by ccstan99**
The GUI (https://nlp.stampy.ai/) will return questions of all status, allowing editors to check for semantically similar questions that are not yet published. Short term solution to https://github.com/StampyAI/stampy-ui/issues/250.

Couple other minor fixes will be noted elsewhere.

### CODE REVIEW

File: src/stampy_nlp/utilities/coda_utils.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def fetch_all_rows():
     headers: object = {'Authorization': f'Bearer {get_coda_token()}'}
     params: object = {
         'useColumnNames': True,
-        'limit': 1000
+        'limit': 10000
```
Comment (ccstan99) [inline src/stampy_nlp/utilities/coda_utils.py:28]: We're nearing the 1000 limit. Increasing it by 10x. Maybe there's a better way to do this?

File: src/stampy_nlp/utilities/coda_utils.py [hunk 2]
```diff
@@ -71,6 +77,8 @@ def get_df_data(is_similar = lambda n1, n2: n1 == n2, delete_all: bool = False):
                     'url': url,
                 })
 
+    df_similar = pd.DataFrame(similar_alternates)
+    df_similar.to_csv("similar_alternates.tsv", sep='\t', index=False)
```
Comment (ccstan99) [inline src/stampy_nlp/utilities/coda_utils.py:81]: Adding a TSV file of skipped similar alternative phrasings for my own record. Might publish it eventually if it gets too large. Someone has been getting a little overly zealous with the list:

I’d like to get deeper into the AI alignment literature. Where should I start?
I’d like to delve deeper into the AI alignment literature. Where should I look?
I’d like to delve deeper into the AI alignment literature. Where should I start?
I’d like to take a deep dive into the AI alignment literature. Where should I look?
I’d like to take a deep dive into the AI alignment literature. Where should I start?
What’s a good starting point for exploring the AI alignment literature?
What’s a good starting point for exploring the alignment literature?

File: src/stampy_nlp/utilities/coda_utils.py (hunk 1 above)
Comment (mruwnik) [reply src/stampy_nlp/utilities/coda_utils.py:28]: yup. Pagination, which the following lines do :) Coda ignores anything over 500, anyway

**Review by mruwnik — APPROVED** [review]

File: src/stampy_nlp/utilities/coda_utils.py (hunk 1 above)
Comment (ccstan99) [reply src/stampy_nlp/utilities/coda_utils.py:28]: Should I just remove that line entirely then?

Comment (mruwnik) [reply src/stampy_nlp/utilities/coda_utils.py:28]: might as well

**Pull request merged**